frontend/tsconfig.json

```diff
@@ -3,24 +3,31 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
-    "strict": true,
+    "strict": false,
     "noEmit": true,
-    "esModuleInterop": true,
+    "incremental": true,
```

Comment:
what does this do?

Reply:
i think one of the tutorials i followed enabled this? it makes it so that only the files that are changed are recompiled